### PR TITLE
Update gradle to 5.6.1

### PR DESCRIPTION
**Pull request by m273d15**
* Updated to the latest gradle version. 
The changes are created by the command:
`./gradlew wrapper --gradle-version=5.6.1`
* Update the gradle scan plugin to the latest version

### CODE REVIEW

**Comment by srossbach** [comment]
Do you upgrade just because a new version was released or is there something specific you would like to resolve using a newer gradle version ?

**Comment by m273d15** [comment]
I updated just to use the newer version.
Sooner or later I will adopt some new syntax which will replace the old syntax (#407).

File: gradlew [hunk 1]
```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS='"-Xmx64m"'
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```
Comment (srossbach) [inline gradlew]: Just wondering if this is a bit too less memory. Have you checked garbage collector times ?

Comment (m273d15) [reply gradlew]: The gradle [docs](https://docs.gradle.org/current/userguide/build_environment.html#sec:configuring_jvm_memory) say:
> The JAVA_OPTS environment variable controls the command line client, which is only used to display console output. It defaults to -Xmx64m

**Pull request merged**